docker/ubuntu_20.04-dev.dockerfile

```diff
@@ -61,105 +59,97 @@ RUN if [ "$build_hdf5" != "NO" ]; then \
+    if [ "$build_hdf5" != "NO" ] ; \ 
+    then \
+            export MOAB_HDF5_ARGS="-DHDF5_ROOT=$HDF5_INSTALL_PATH"; \
+    fi \
+    && cd $HOME/opt \
+    && mkdir moab \
+    && cd moab \
+    && git clone --depth 1 --single-branch -b 5.3.0 https://bitbucket.org/fathomteam/moab \
+    && cd moab \
+    && mkdir build \
+    && cd build \
+    && ls ..\
+    # build/install shared lib
+    && cmake .. \
+            ${PYMOAB_FLAG} \
+            -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH \
+            -DENABLE_HDF5=ON $MOAB_HDF5_ARGS \
+            -DBUILD_SHARED_LIBS=ON \
+            -DENABLE_BLASLAPACK=OFF \
+            -DENABLE_FORTRAN=OFF \
+    && make -j 3 \
+    && make install \
+    && cd .. \
+    && rm -rf moab ;
 
 # put MOAB on the path
 ENV LD_LIBRARY_PATH $HOME/opt/moab/lib:$LD_LIBRARY_PATH
 ENV LIBRARY_PATH $HOME/opt/moab/lib:$LIBRARY_PATH
 ENV PYTHONPATH=$HOME/opt/moab/lib/python${py_version}/site-packages/
 
+FROM moab AS dagmc
 # build/install DAGMC
-ARG build_dagmc="NO"
 ENV INSTALL_PATH=$HOME/opt/dagmc
-RUN if [ "$build_dagmc" = "YES" ]; then \
-        cd /root \
-        && git clone --depth 1 --branch stable https://github.com/svalinn/DAGMC.git \
-        && cd DAGMC \
-        && mkdir bld \
-        && cd bld \
-        && cmake .. -DMOAB_DIR=$HOME/opt/moab \
-                 -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH \
-                 -DBUILD_STATIC_LIBS=OFF \
-                 -DBUILD_UWUW=OFF \
-                 -DBUILD_TALLY=OFF \
-                 -DBUILD_MAKE_WATERTIGHT=OFF \
-                 -DBUILD_OVERLAP_CHECK=OFF \
-                 -DBUILD_TESTS=OFF \
-        && make -j 3\
-        && make install \
-        && cd ../.. \
-        && rm -rf DAGMC; \
-    fi
+RUN cd /root \
+    && git clone --depth 1 --branch stable https://github.com/svalinn/DAGMC.git \
+    && cd DAGMC \
+    && mkdir bld \
+    && cd bld \
+    && cmake .. -DMOAB_DIR=$HOME/opt/moab \
+                -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH \
+                -DBUILD_STATIC_LIBS=OFF \
+                -DBUILD_UWUW=OFF \
+                -DBUILD_TALLY=OFF \
+                -DBUILD_MAKE_WATERTIGHT=OFF \
+                -DBUILD_OVERLAP_CHECK=OFF \
+                -DBUILD_TESTS=OFF \
+    && make -j 3\
+    && make install \
+    && cd ../.. \
+    && rm -rf DAGMC
 
-ARG build_pyne=YES
 # Build/Install PyNE
-RUN if [ "$build_pyne" = "YES" ]; then \
-        export PYNE_HDF5_ARGS="" ;\
-        if [ "$build_hdf5" != "NO" ]; then \
-              export PYNE_HDF5_ARGS="--hdf5 $HDF5_INSTALL_PATH" ; \
-        fi \
-        && cd $HOME/opt \
-        && git clone -b develop --single-branch https://github.com/pyne/pyne.git \
-        && cd pyne \
-        && python setup.py install --user \
-                                    --moab $HOME/opt/moab --dagmc $HOME/opt/dagmc \
-                                    $PYNE_HDF5_ARGS \
-                                    --clean -j 3; \
-    fi
+RUN export PYNE_HDF5_ARGS="" ;\
```

Comment:
```suggestion
FROM dagmc AS pyne
ARG build_hdf5

RUN export PYNE_HDF5_ARGS="" ;\
```

Reply:
Should I change the stages in the multistage build action in `docker_publish.yml` to reflect the addition of this new `pyne` stage? Also, should the `openmc` stage still be based on `dagmc` or should it now be based on `pyne`?

Reply:
I think the openmc should be based on dagmc and not on PyNE